### PR TITLE
qoriq: Expand generic subtarget description

### PR DESCRIPTION
**Pull request by ghost**
The NXP QorIQ AMP Series Power Architecture Processing Platforms are now since launch (cira 2010) more commonly referred to as T-Series(T1-T5).

### CODE REVIEW

File: target/linux/qoriq/generic/target.mk [hunk 1]
```diff
@@ -1,3 +1,5 @@
 define Target/Description
-        Build firmware images for NXP QorIQ boards in the AMP series.
+        Build firmware images for all NXP QorIQ boards in the
```
Comment (stintel) [inline target/linux/qoriq/generic/target.mk:2]: I would remove the all here as it's misleading, we currently only support M300?

**Review by stintel — CHANGES_REQUESTED** [review]

File: target/linux/qoriq/generic/target.mk (hunk 1 above)
Comment (mattytap) [reply target/linux/qoriq/generic/target.mk:2]: hi stintel you are correct. Thank you. I will change

**Comment by stintel** [comment]
Now please squash the commits and force push to the branch.

**Comment by stintel** [comment]
Now there are 8 commits in this PR ... not good :) Find me on IRC and I'll guide you through it.

**Comment by mattytap** [comment]
Hi Stijn 
Yes, sorry about this, I had a git nightmare last night. Should still have created a separate local branch for this PR even though it was a trivial change. Will irc later. Thanks for offer to help. 

Best Wishes
Matt Fawcett

On 24 Dec 2021, at 02:24, Stijn Tintel ***@***.***> wrote:

﻿
Now there are 8 commits in this PR ... not good :) Find me on IRC and I'll guide you through it.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Review by stintel — APPROVED** [review]

**Comment by stintel** [comment]
Merged, thanks!

**Pull request closed without merging**